spec.bs

```diff
@@ -5108,6 +5146,15 @@ An <dfn>ad size</dfn> is the width and height of an ad.
   1. Return |adSize|.
 </div>
 
+An <dfn>ad keyword replacement</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="ad keyword replacement">
+  : <dfn>match</dfn>
+  :: A [=string=], that is a [=code unit substring=] of {{AuctionAd/renderURL}}. Must be in the format `${<match>}` or `%%<match>%%`

```

Comment:
Maybe something similar to what you have in code:
`A [=string=], represents the match within the {{AuctionAd/renderURL}} that we are looking to replace with [=ad keyword replacement/replacement=]`.

Also, maybe `${...}` or `%%...%%`. The term "match" inside looks a little confusing as the field's name is "match".

Reply:
Done.